dockerfiles/Dockerfile.tensorrt

```diff
@@ -5,7 +5,7 @@
 # Dockerfile to run ONNXRuntime with TensorRT integration
 
 # nVidia TensorRT Base Image
-FROM nvcr.io/nvidia/tensorrt:20.12-py3
+FROM nvcr.io/nvidia/tensorrt:20.09-py3
```

Comment:
we'll need a task for longer term how to address the 20.12 issue, because eventually we would want to update this Dockerfile to newer tensorrt base image version.

Reply:
Will upgrading Miniconda3-4.5.11 to Miniconda3-4.7.12 solve the issue? We've claimed ORT can support TRT7.2.2. If we roll back to 20.09, only TRT7.1 is supported in the container.

Reply:
I see you've upgraded miniconda. What's the problem of TRT 20.12 container? is it about GLIBCXX.3.4.26?

Reply:
Yes, see this item https://msdata.visualstudio.com/Vienna/_boards/board/t/ONNX%20Inference/Features/?workitem=1123564